pom.xml

```diff
@@ -1056,6 +1094,81 @@
     </profile>
 
     <profile>
+      <id>jacoco</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.7.5.201505241946</version>
+            <configuration>
+              <haltOnFailure>true</haltOnFailure>
```

Comment:
either this ``haltOnFailure`` is useless or one in execution with id ``default-check``